tests/unit/src/test/scala/tests/markdown/BaseMarkdownSuite.scala

```diff
@@ -61,4 +61,23 @@ abstract class BaseMarkdownSuite extends org.scalatest.FunSuite with DiffAsserti
       assertNoDiffOrPrintExpected(obtained, expected)
     }
   }
+
+  def checkWithSettings(
```

Comment:
Overriding the settings doesn't allow to assign different settings to different tests.

Passing `settings` to `check` doesn't work because `context` has already been initialized at that point.

I can refactor the `BaseMarkdownSuite` to be more stateless, but that's a bigger change.

Reply:
I would have created a `ScastieDarkSuite` to work around the evil state 😜 

I refactored `BaseMarkdownSuite` to be state-less so that it's possible to use different settings in the same suite, just for you 😘